static/app/views/dashboardsV2/widgetBuilder/widgetBuilder.tsx

```diff
@@ -219,16 +230,22 @@ function WidgetBuilder({
       };
     }
 
+    const visualization =
+      widgetBuilderNewDesign && widgetToBeUpdated.displayType === DisplayType.TOP_N
```

Comment:
Why do we need this kind of check here and on line 203? They're converting TOP_N queries to tables when the builder renders for the first time, right? Is that necessary?

Reply:
yes because now with the new fields, we no longer want to display top 5 events in the display options